File: app/src/main/java/com/texon/engineeringsmartbook/View/SuccessfullyEnrolled.java
package com.texon.engineeringsmartbook.View;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageButton;

import com.texon.engineeringsmartbook.R;

public class SuccessfullyEnrolled extends AppCompatActivity {

    ImageButton btnBackMenu, btnNotificationsMenu, bntHomeMenu, btnProfileMenu, btnScanMenu;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_successfully_enrolled);

        btnBackMenu = findViewById(R.id.btnBackMenu);
        btnNotificationsMenu = findViewById(R.id.btnNotificationsMenu);
        bntHomeMenu = findViewById(R.id.btnHomeMenu);
        btnProfileMenu = findViewById(R.id.btnProfileMenu);
        btnScanMenu = findViewById(R.id.btnScannerMenu);



        btnBackMenu.setOnClickListener(v -> {
            Intent intent = new Intent(getApplicationContext(), BookEnrollment.class);
            startActivity(intent);
            finish();
        });
    }

    @Override
    public void onBackPressed() {
        Intent intent = new Intent(getApplicationContext(), BookEnrollment.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/texon/engineeringsmartbook/ui/main/view/SuccessfullyEnrolled.kt
package com.texon.engineeringsmartbook.ui.main.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.view.WindowManager
import com.texon.engineeringsmartbook.databinding.ActivitySuccessfullyEnrolledBinding
import kotlinx.coroutines.DelicateCoroutinesApi

@DelicateCoroutinesApi
class SuccessfullyEnrolled : AppCompatActivity() {

    private lateinit var binding: ActivitySuccessfullyEnrolledBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivitySuccessfullyEnrolledBinding.inflate(layoutInflater)
        setContentView(binding.root)
        window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN)

        binding.titleBar.btnBackMenu.setOnClickListener{
            val intent = Intent(applicationContext, BookEnrollment::class.java)
            startActivity(intent)
            finish()
        }
    }

    override fun onBackPressed() {
        val intent = Intent(applicationContext, BookEnrollment::class.java)
        startActivity(intent)
        finish()
    }
}